### PR TITLE
Added: Cards component

### PR DESCRIPTION
**Pull request by pratik-kumar-sahu**
- Added Cards component.
- Live link --> https://deploy-preview-5--fuzz-ui.netlify.app/pages/docs.html

### CODE REVIEW

File: styles/components/button.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.btn-txt:hover {
+    padding-bottom: 3px;
+    border-bottom: 2px solid var(--color-black);
+}
+
+.btn-primary-solid {
+    color: var(--color-white);
+    background-color: #0ea5e9;
+    border: 2px solid #0ea5e9;
```
Comment (DivyanshSareen) [inline styles/components/button.css]: use var for all the colors. Overall code looks good

Comment (pratik-kumar-sahu) [reply styles/components/button.css]: Sure, thanks

File: styles/_components.css [hunk 2]
```diff
@@ -94,3 +91,71 @@
...
+    background-color: #2b3e50;
+    overflow: auto;
+    border-radius: 1rem;
+}
+
+.iframe {
+    width: 100%;
+    height: auto;
+    overflow: hidden;
+    /* border: 1px solid var(--color-background); */
```
Comment (AtulHande03a) [inline styles/_components.css]: you can remove this

File: styles/components/button.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+}
+
+.btn-primary-solid:hover {
+    background-color: #0d7caf;
+}
+
+.btn-primary-outline {
+    color: #0ea5e9;
```
Comment (AtulHande03a) [inline styles/components/button.css]: you can define variables for the hex colors and use them

**Review by AtulHande03a — APPROVED** [review]
Review: remove the unwanted comments , use variables instead of the hex colors .. rest looks good.

File: styles/_components.css (hunk 2 above)
Comment (pratik-kumar-sahu) [reply styles/_components.css]: okay

File: styles/components/button.css (hunk 3 above)
Comment (pratik-kumar-sahu) [reply styles/components/button.css]: Yes sure thanks

**Pull request merged**